dokka-runners/dokka-gradle-plugin/src/testFixtures/kotlin/files.kt

```diff
@@ -21,3 +24,18 @@ fun Path.listRelativePathsMatching(predicate: (Path) -> Boolean): List<String> {
         .toList()
         .sorted()
 }
+
+/**
+ * Create a temporary directory.
+ *
+ * Kotest will attempt to delete the file after the current spec has completed.
+ *
+ * (@see [io.kotest.engine.spec.tempdir], but this returns a [Path], not a [java.io.File].)
+ */
+fun TestConfiguration.tempDir(prefix: String? = null): Path {
```

Comment:
Why not just use `io.kotest.engine.spec.tempdir.toPath()`? :thinking:

Reply:
I find having `toPath()` everywhere is repetitive and distracting, so it's nicer to have a convenient shortcut.

Reply:
You could still have shortcut, but under-the-hood just invoke kotest method and convert it to path.

It is more about not reinventing the wheel :slightly_smiling_face: